server/server_test.go

```diff
@@ -170,7 +170,20 @@ func TestUpdateExtensions(t *testing.T) {
 
 	// Single extension same version
 	requestBody = lightThemeExtension("1.0.0")
-	expectedResponse = "<response protocol=\"3.1\" server=\"prod\"></response>"
+	expectedResponse = `<response protocol="3.1" server="prod">
+    <app appid="ldimlcelhnjgpjjemdjokpgeeikdinbm">
```

Comment:
Do you know, does the Google component update server return the same when there is no update?

Reply:
Yes the response is similar. :
```
<response protocol=\"3.1\" server=\"prod\"><daystart elapsed_days=\"4404\" elapsed_seconds=\"46144\"/><app appid=\"giekcmmlnklenlaomppkphknjmnnpneh\" cohort=\"1:j5l:\" cohorthint=\"Auto\" cohortname=\"Auto\" status=\"ok\"><updatecheck status=\"noupdate\"/><ping status=\"ok\"/></app></response>
```

The response from `go-updater` does have some extraneous fields but the fields are ignored on the client side if the status is `noupdate`. The component updater code looks for the `update-check` status to display the result:
https://cs.chromium.org/chromium/src/components/update_client/component.cc?g=0&l=511